src/routes/router.js

```diff
@@ -0,0 +1,5 @@
+const router = require('express').Router();
+
+router.get('/', require('./home.js'))
```

Comment:
I think this is fine for the home route but for future routes possibly:

```js
router.route('places')
      .get(controllerhere)
      .post(contollerher)
```

Looks a lot better/is standard